F3FChrono/gui/MainUiController.py

```diff
@@ -123,6 +136,7 @@ def next_action(self):
                                                                     self.chrono, self.chronoHard.getPenalty(), True)
 
             self.chronoHard.reset()
+            self.chrono.reset()
```

Comment:
Warning : I think that we should create a new Chrono object here. Otherwise, modifications made to this object may be applied also to previous pilots !

Reply:
I create a copy of chrono value in a new chrono class in Handle_terminated_flight.
I could write it as you wish. 
I check this afternoon